README.md

```diff
@@ -315,6 +315,18 @@ setVerbose(True)
 # turn off verbose mode
 setVerbose(False)
 ```
+### Bypassing merge task 
+
+Skipping the merge task is useful when the tasks results don't need to be merge into a list. To bypass the merge task, you can pass the *enableMerge* flag to the foreach object:
+
+```R
+# Enable merge task
+foreach(i = 1:3, .options.azure = list(enableMerge = TRUE))
+
+# Disable merge task
+foreach(i = 1:3, .options.azure = list(enableMerge = FALSE))
+```
+Note: User defined functions for the merge task is on our list of features that we are planning on doing.

```

Comment:
merge and the .combine argument are different.

what we have today is that the merge tasks takes the results (usually a list) and puts them into a list: [<list of results1>, <list of results2>, etc] - this happens on the 'merge-node'

Then, you can apply a combine on that list to do something to that list. This combine happens on the local node. for example, .combine=c will just flatten the list (on the local node).